app/locales/en.json

```diff
@@ -102,13 +102,13 @@
     "logging_inactive": "Location Inactive"
   },
   "history": {
-    "no_exposure": "No exposure",
+    "no_exposure": "No known",
     "possible_exposure_para": "It is possible you were in contact with or close to someone who tested positive for COVID-19",
-    "possible_exposure": "Possible exposure",
+    "possible_exposure": "Possible",
```

Comment:
Given the context of what else is now on this/these screen(s), is there a way to make sure that the words Possible and No known are enough to convey the meaning to an average user?
Maybe a screenshot of these screens and show it to a few multilingual people or the ones not immediately familiar with the screens?

Reply:
I attached a picture of the small screen.